x-pack/platform/plugins/shared/index_management/public/application/sections/home/index_list/details_page/convert_to_lookup_index_modal/convert_to_lookup_index_modal_container.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { useConvertIndexToLookup } from '../../../../../hooks/use_convert_index_to_lookup/use_convert_index_to_lookup';
+import { ConvertToLookupIndexModal } from './convert_to_lookup_index_modal';
+
+import { notificationService } from '../../../../../services/notification';
+
+export const ConvertToLookupIndexModalContainer = ({
+  onCloseModal,
+  onSuccess,
+  sourceIndexName,
+}: {
+  onCloseModal: () => void;
+  onSuccess: (lookupIndexName: string) => void;
+  sourceIndexName: string;
+}) => {
+  const { isConverting, errorMessage, convert } = useConvertIndexToLookup({
+    sourceIndexName,
+    onSuccess,
+    onClose: onCloseModal,
+  });
+
+  const onCloseModalWithInfoToast = () => {
+    if (isConverting) {
+      notificationService.showInfoToast(
+        i18n.translate(
+          'xpack.idxMgmt.convertToLookupIndexAction.indexConversionInProgressToastTitle',
+          {
+            defaultMessage: 'Index conversion in progress',
+          }
+        ),
+        i18n.translate(
+          'xpack.idxMgmt.convertToLookupIndexAction.indexConversionInProgressToastMessage',
+          {
+            defaultMessage: 'The conversion operation is still in progress for {sourceIndexName}.',
```

Comment:
```suggestion
            defaultMessage: '{sourceIndexName} conversion into a lookup index is still in progress.',
```
doesn't add that much value, is it mandatory to have a title + body? We could combine the information into just a title or just the body.

Reply:
Updated in 9d77bb8